src/decenc_aec.c

```diff
@@ -0,0 +1,152 @@
+/** @file
+ * @brief Decode and encode functions for AEC compression.
+ * stream
+ * @author Eric Engle @date 2023-10-16
+ *
+ * Program History Log
+ * Date | Programmer | Comments
+ * -----|------------|---------
+ * 2023-10-16 | Engle | Initial
+ */
+
+#include <stdio.h>
+#include <stdint.h>
+#include <libaec.h>
+#include "grib2_int.h"
+
+/**
+ * This Function decodes an AEC code stream specified in the
+ * [CCSDS 121.0-B-3 Blue Book](https://public.ccsds.org/Pubs/121x0b3.pdf).
+ *
+ * @param cpack Pointer to buffer that holds the input AEC code
+ * stream.
+ * @param len Length (in bytes) of the buffer that holds the input
+ * AEC code stream.
+ * @param nbits CCSDS bits per sample.
+ * @param flags CCSDS compression options mask.
+ * @param block_size CCSDS block size.
+ * @param rsi CCSDS reference sample interval.
+ * @param cfld Pointer to output buffer from the AEC decoder.
+ * @param cfldlen length of output buffer.
+ *
+ * @return
+ * - >0 Length of data from AEC decoder
+ * - 0 Successful decode (AEC_OK)
+ * - -1 AEC_CONF_ERROR
+ * - -2 AEC_STREAM_ERROR
+ * - -3 AEC_DATA_ERROR
+ * - -4 AEC_MEM_ERROR
+ * - -5 AEC_RSI_OFFSETS_ERROR
+ *
+ * @author Eric Engle @data 2023-10-16
```

Comment:
Should be "date" not "data", this is what is causing your developer build to fail in CI...

Reply:
Fixed with [38bef85](https://github.com/NOAA-EMC/NCEPLIBS-g2c/pull/449/commits/38bef85423c039e7572e565e71154bd0953cf562)